src/main/ngapp/src/app/register/register.component.html

```diff
@@ -102,7 +16,7 @@ <h4>Registration</h4>
         <nb-select placeholder="Select Showcase" ngModel #language="ngModel" name="language" id="language" required [status]="language.dirty? (language.invalid? 'danger': 'success'): 'basic'">
             <nb-option value="en">English</nb-option>
```

Comment:
Seeing this list showing up in two files. Consider move the language into a file and generate options dynaimcally.